src/pyramid_session_redis/__init__.py

```diff
@@ -122,8 +122,10 @@ def RedisSessionFactory(
     cookie_signer=None,
     socket_timeout=None,
     connection_pool=None,
-    charset="utf-8",
-    errors="strict",
+    charset=None,
+    errors=None,
+    encoding=None,
+    encoding_errors=None,
     unix_socket_path=None,
```

Comment:
Do you think it is worth prefixing the new arguments with "redis_", since they are passthrough?

 i.e.:

* redis_encoding=None
* redis_encoding_errors=None